db/migrate/20210224230319_add_smal_image_url_to_ads.rb

```diff
@@ -1,5 +1,5 @@
 class AddSmalImageUrlToAds < ActiveRecord::Migration[5.0]
   def change
-    add_column :ads, :small_image_url, :string, null: false
+    add_column :ads, :small_image_url, :string, default: '', null: false
```

Comment:
you could've done: `git commit --fixup` to amend the original commit with the migration, as long as it is not merged into main master, we can do that

Reply:
That's a good point! I will rebase to squash some commit